### PR TITLE
deleted state in useSyncProps

### PR DESCRIPTION
**Pull request by daurenSab**
## Description
Closes https://github.com/epicmaxco/vuestic-ui/issues/1602

## Markup:

<details>

```vue
// Your code
```
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-rating/components/VaRatingItem/VaRatingItem.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default defineComponent({
 
   setup (props, { emit }) {
     const rootEl = ref<HTMLElement>()
-    const [modelValue] = useSyncProp('modelValue', props, emit, RatingValue.EMPTY)
```
Comment (asvae) [inline packages/ui/src/components/va-rating/components/VaRatingItem/VaRatingItem.vue:55]: Why default value disappeared?

**Comment by daurenSab** [comment]
Closes #1602

**Comment by asvae** [comment]
Apart from that - looks good! 👍

**Review by asvae — APPROVED** [review]

**Comment by asvae** [comment]
Thanks!

**Pull request merged**